src/ui/public/number_list/number_list.js

```diff
@@ -88,7 +88,17 @@ uiModules
             const list = self.getList();
             if (!list) return;
 
-            list.push(_.last(list) + 1);
+            function getNext(last, max) {
+              const next = last + 1;
+              if (next < max) {
```

Comment:
Question: Would it be better to do `next <= max` since _technically_ it is valid to go up to `100`? I don't have strong feelings either way.

Reply:
The directive is only used in percentile_ranks and percentiles metric aggregation editors. In those cases, I see wanting to define 99.9, 99.99, and so on as more likely then defining a bucket of 100.